### ISSUE TITLE
Cannot find chrome.app in OSX

### ISSUE THREAD

**Issue by ivogallego** [karma-runner/karma-chrome-launcher#53]
In the latest version of the app (0.1.12) the launcher is trying to find the Chrome executable in the home directory (~/Applications/Google Chrome.app) instead of the common one (/Applications/Google Chrome.app)

**Comment by provegard** [comment]
I'm getting this on a Windows machine as well:

`ENOENT, no such file or directory 'C:\Users\TeamCity\AppData\Local\Google\Chrome\Application'`

Breaks my build server :-(

This is version 0.1.11.

**Comment by ivogallego** [comment]
I was researching on how to fetch only stable dependencies with bower, but it looks like semver doesn't allow that... shouldn't the beta versions be released in a different way?

**Comment by provegard** [comment]
I downgraded to 0.1.10 and the problem went away.

**Comment by Dignifiedquire** [comment]
Actually it should look in both, and the `ENONET` issue should be fixed via #52. Could you check the master version if that works for you please?

**Comment by fzaninotto** [comment]
:+1: same problem on my side, please revert the regression.

**Comment by Dignifiedquire** [comment]
can anyone please provide a reliable way to reproduce this? For me everything is running fine :/

**Comment by valentinomiazzo** [comment]
Same on my Windows 7. karma-chrome-launcher 0.1.12.
Version 0.1.8 works on the same PC.

I have done some debugging and the problem is the version of Node.js.
Basically `fs.accessSync` is not supported, the code shallows the exception and exhausts all the possible paths.

```javascript
function getChromeExe(chromeDirName) {
  ...
  for (i = 0; i < prefixes.length; i++) {
    prefix = prefixes[i];
    try {
      windowsChromeDirectory = path.join(prefix, suffix);
      fs.accessSync(windowsChromeDirectory);
      return windowsChromeDirectory;
    } catch (e) {
      console.log('%s - %s', windowsChromeDirectory, e);  //<---- added by me
    }
  }

  return windowsChromeDirectory;
}
```

Generates

```
C:\Users\vm\AppData\Local\Google\Chrome\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
C:\Program Files\Google\Chrome\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
C:\Program Files (x86)\Google\Chrome\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
C:\Users\vm\AppData\Local\Google\Chrome SxS\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
C:\Program Files\Google\Chrome SxS\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
C:\Program Files (x86)\Google\Chrome SxS\Application\chrome.exe - TypeError: Object #<Object> has no method 'accessSync'
```

**Comment by Dignifiedquire** [comment]
@valentinomiazzo thanks for the investigation, what node version are you seeing this with?

**Comment by valentinomiazzo** [comment]
```
$ node --version
v0.10.13
```

**Comment by Dignifiedquire** [comment]
A fix should be available as `0.2.0`.
